scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -4,50 +4,39 @@ import "GvsQuickstartVcfIntegration.wdl" as QuickstartVcfIntegration
 import "GvsQuickstartHailIntegration.wdl" as QuickstartHailIntegration
 import "GvsUtils.wdl" as Utils
 
-task FilterIntervalListChromosomes {
-    input {
-        File full_interval_list
-        Array[String]+ chromosomes
-    }
-    command <<<
-        # Prepend date, time and pwd to xtrace log entries.
-        PS4='\D{+%F %T} \w $ '
-        set -o errexit -o nounset -o pipefail -o xtrace
-
-        python3 /app/filter_interval_list_chromosomes.py --input-interval-list ~{full_interval_list} \
-            --output-interval-list "filtered.interval_list" --chromosome ~{sep=' --chromosome ' chromosomes}
-    >>>
-    runtime {
-        docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-07-24-alpine-bd6b9d62e"
-    }
-    output {
-        File out = "filtered.interval_list"
-    }
-}
-
 workflow GvsQuickstartIntegration {
     input {
         String branch_name
+        Boolean use_default_dockers = false
         Boolean run_vcf_integration = true
         Boolean run_hail_integration = true
+        Boolean run_exome_integration = true
         String? sample_id_column_name ## Note that a column WILL exist that is the <entity>_id from the table name. However, some users will want to specify an alternate column for the sample_name during ingest
         String? vcf_files_column_name
         String? vcf_index_files_column_name
         String? sample_set_name ## NOTE: currently we only allow the loading of one sample set at a time
     }
 
-    File full_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
-    File expected_output_prefix = "gs://gvs-internal-quickstart/integration/2023-07-19-quicker/"
+    File full_wgs_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
+    File full_exome_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/bge_exome_calling_regions.v1.1.interval_list"
+    File expected_output_prefix = "gs://gvs-internal-quickstart/integration/2023-07-25-quicker/"
+
+    # WDL 1.0 trick to set a variable ('none') to be undefined.
+    if (false) {
+        File? none = ""
+    }
```

Comment:
I very much liked the idea of always having *some* exercise of no-GATK-override in this test, but probably that should be worked into my PR rather than this one.

Reply:
Yeah. Would like to get this merged before I head out on vacay at the end of this week.